distribution/appserver-home/RUNNING.txt

```diff
@@ -182,18 +143,10 @@ launches Java directly and does not use the script files.
 
   On Windows:
 
-      %CATALINA_HOME%\bin\startup.bat
-
-    or
-
       %CATALINA_HOME%\bin\catalina.bat start
 
   On *nix:
 
-      $CATALINA_HOME/bin/startup.sh
-
-    or
-
       $CATALINA_HOME/bin/catalina.sh start
 
 (4.2) After startup, the default web applications included with Application Server will be

```

Comment:
We better refer to Application Server as "WSO2 Application Server", shall we change this in all places? WDYT?